client/src/components/ElementEditor/Element.js

```diff
@@ -21,6 +21,10 @@ import { elementDragSource, isOverTop } from 'lib/dragHelpers';
  * of an element's details when used in the CMS, including ID, Title and Summary.
  */
 class Element extends Component {
+  static getDerivedStateFromError() {
+    return { editingError: true };
```

Comment:
`editingError` feels like it's caused by users editing the stuff in the element however, error boundaries don't catch errors in event handlers. Should we rename it to something like `childrenRenderingError`?

Reply:
Tweaked :)